src/seedu/addressbook/data/person/AddressBlock.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+public class AddressBlock {
+	private String value;
+	private boolean isPrivate;
```

Comment:
Is this necessary?  Will this property be different for different address components?